build_overrides/webnn_features.gni

```diff
@@ -33,4 +33,5 @@ declare_args() {
   webnn_enable_null = true
 
   webnn_enable_wire = false
+  webnn_enable_gpu_buffer = false
```

Comment:
What's the dependency of this one? Does it require enable dml?

Reply:
I mean it would be good to document it in the readme.

Reply:
How about documenting it in building Chromium which will be used in there?

Reply:
It sounds good.